### ISSUE TITLE
after exiting game dash studio doesn't show idle screen

### ISSUE THREAD

**Issue by Meekstaaa** [SHWotever/SimHub#426]
Since latest update (I think) when exiting out it sometime doesn't go back to Idle screen.

Idle is set to first on the order of the pages

**Comment by SHWotever** [comment]
Hi!

Is it on a specific game ? I could not reproduce it with the standard dash.

NB: on ACC particularly it's the case due to shared memory not reporting correctly race/game exit. I tried lot of workarounds but it all ended causing issues so I gave up with this until they fix the data.

**Comment by Meekstaaa** [comment]
Hey,

I’m using iracing. Noticed it on two dashboards.

Many thanks

David Meek
Sent from my iPhone

On 6 Oct 2019, at 19:38, Wotever <notifications@github.com> wrote:

﻿

Hi!

Is it on a specific game ? I could not reproduce it with the standard dash.

NB: on ACC particularly it's the case due to shared memory not reporting correctly race/game exit. I tried lot of workarounds but it all ended causing issues so I gave up with this until they fix the data.
